src/CodeGen_Hexagon.cpp

```diff
@@ -1003,6 +1003,18 @@ const HvxIntrinsic intrinsic_wrappers[] = {
      "acc_add_2mpy.vw.vh.b",
      {i32v1, i16v1, i16},
      HvxIntrinsic::BroadcastScalarsToWords},
+    {MAKE_ID_PAIR(Intrinsic::hexagon_V6_vdmpyhsat),
```

Comment:
MAKE_ID_PAIR was removed when hvx_64 support was removed. This branch is a wee bit out of date :-)

Reply:
Thanks Steven for pointing this. Fixed it